setup.py

```diff
@@ -27,15 +33,19 @@ def get_readme():
     packages=find_packages(),
     entry_points={
         'console_scripts': [
-            'argparse-manpage=build_manpages.cli:main',
+            'argparse-manpage=argparse_manpage.cli:main',
         ],
     },
     description='Build manual page from python\'s ArgumentParser object.',
     long_description=get_readme(),
     long_description_content_type='text/markdown',
     cmdclass={
         'build_manpages': build_manpages,
-        'build_py': get_build_py_cmd(build_py),
-        'install': get_install_cmd(install),
+        'build_py': get_build_py_cmd(),
+        'install': get_install_cmd(),
+    },
+    # TODO: switch to extras_require once we stop EL7 support
+    dependencies={
+        "build-manpages": ['setuptools;python_version>="3.12"'],
     },
```

Comment:
I have never seen this syntax before. Is this something I don't know, or a mistake?

Reply:
Probably a mistake!